src/ChronosInterval.php

```diff
@@ -166,298 +74,60 @@ public function __construct(
      * @return static
      */
     public static function create(
-        ?int $years = 1,
+        ?int $years = null,
         ?int $months = null,
         ?int $weeks = null,
         ?int $days = null,
         ?int $hours = null,
         ?int $minutes = null,
         ?int $seconds = null,
-        ?int $microseconds = null
+        ?int $microseconds = null,
     ): static {
```

Comment:
Nice! I'm glad to see we're keeping a method that provides this more ergonomic signature over the hard to remember interval string format that PHP uses.